src/types/hexColorCode.ts

```diff
@@ -0,0 +1,12 @@
+import {StrongType, makeStrong} from '../strong-type';
+
+import {STRules} from '../rules';
+
+export type StrongHexCode = StrongType<string>;
```

Comment:
For consistency and to better describe the type instances of HexCode should be HexColorCode.

- [ ] Change `StrongHexCode` to `StrongHexColorCode`.